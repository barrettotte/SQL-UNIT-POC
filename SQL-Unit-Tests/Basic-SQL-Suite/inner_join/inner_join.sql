-- This is a simple test for an INNER JOIN statement --

INSERT INTO developers (username, fav_language) VALUES ('barrettotte', 'Python');
INSERT INTO developers (username, fav_language) VALUES ('firstlast', 'Java');
INSERT INTO developers (username, fav_language) VALUES ('helloworld123', 'JavaScript');
INSERT INTO developers (username, fav_language) VALUES ('someguy1', 'C++');


SELECT DISTINCT u.first_name, u.last_name, d.fav_language
FROM [BARRETT_TEST].[dbo].[developers] AS d
INNER JOIN [BARRETT_TEST].[dbo].[users] AS u 
	ON u.username = d.username;