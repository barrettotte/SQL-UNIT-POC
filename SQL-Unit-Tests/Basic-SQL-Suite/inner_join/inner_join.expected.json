{
    "success": "true",
    "columns": [
        {
            "name": "first_name",
            "type": "VARCHAR(50)"
        },
        {
            "name": "last_name",
            "type": "VARCHAR(50)"
        },
        {
            "name": "fav_language",
            "type": "VARCHAR(50)"
        }
    ],
    "rows": [
        {
            "first_name": "barrett",
            "last_name": "otte",
            "fav_language": "Python"
        },
        {
            "first_name": "hello",
            "last_name": "world",
            "fav_language": "JavaScript"
        }
    ]
}